js/domUpdates.js

```diff
@@ -1,59 +1,97 @@
-const players = [];
-const clues = [];
-const rounds = [];
-let game;
+const jeopardy = {
+  players: [],
+  clues: [],
+  rounds: [],
+  game: {},
+  categoryNames: {
+    unitedStatesHistory: 'United States History',
+    lifeSciences: 'Life Sciences',
+    publicHealth: 'Public Health',
+    educationJargon: 'Education Jargon',
+    nameThatBoardGame: 'Name That Board Game',
+    americanLiterature: 'American Literature',
+    biographies: 'Biographies',
+    americanCities: 'American Cities',
+    food: 'Food',
+    cableTV: 'Cable TV'
+  }
+};
 
-$('.input--submit').on('click', createPlayers);
-$('.main').on('click', presentClue);
+$('.input--submit').on('click', instantiatePlayers);
 
 function presentClue(event) {
-  console.log(event.target)
-  let colRow = event.target.dataset.colrow;
-  console.log(colRow);
-  let selectedClue = rounds[0].clues[parseInt(colRow)];
-  console.log(selectedClue)
-  alert(selectedClue.question);
+  const id = event.target.dataset.id;
+  const selectedClue = jeopardy.rounds[0].clues[parseInt(id)];
+  if (selectedClue.dailyDouble === true) {
+    alert('DAILY DOUBLE!\n' + selectedClue.question);
+  } else {
+    alert(selectedClue.question);
+  }
+  event.target.innerHTML = '';
+  jeopardy.game.cluesRemaining--;
 }
 
-function createPlayers(event) {
+function instantiatePlayers(event) {
   event.preventDefault();
-  players.push(new Player($('#name-0').val()));
-  players.push(new Player($('#name-1').val()));
-  players.push(new Player($('#name-2').val()));
+  jeopardy.players.push(new Player($('#name-0').val()));
+  jeopardy.players.push(new Player($('#name-1').val()));
+  jeopardy.players.push(new Player($('#name-2').val()));
```

Comment:
Jeopardy is now our global variable that we can reach into to get our classes instead of setting them  all  as individual  global variables.